pkg/audit/manager.go

```diff
@@ -554,7 +537,7 @@ func (am *Manager) skipExcludedNamespace(namespace string) bool {
 
 func (ucloop *updateConstraintLoop) updateConstraintStatus(ctx context.Context, instance *unstructured.Unstructured, auditResults []auditResult, timestamp string, totalViolations int64) error {
 	constraintName := instance.GetName()
-	log.Info("updating constraint status", "constraintName", constraintName)
+	ucloop.log.Info("updating constraint status", "constraintName", constraintName)
```

Comment:
Not sure why we're making `log` a member of the object?

I don't think it breaks anything though.

Reply:
so we can get the audit_id easily for debugging purposes, which is included in `am.log`

Reply:
+1